nixos/modules/system/boot/loader/grub/install-grub.pl

```diff
@@ -286,17 +287,20 @@ sub GrubFs {
     ";
 
     if ($font) {
+	my $gfxpayloadEfi = $gfxpayload ? $gfxpayload : "keep";
```

Comment:
This should be indented

Reply:
yep, will do, I think this might have been a merge artifact, I'll fix it tonight.